namespace-provisioner/about.hbs.md

```diff
@@ -19,7 +19,7 @@ Controller mode
     ![Diagram of Namespace Provisioner controller.](../images/namespace-provisioner-controller.png)
 
     - The list of developer namespaces is managed by the Namespace Provisioner controller using a label selector `apps.tanzu.vmware.com/tap-ns=""`
-    - Namespace Provisioner creates [default resources](reference.md#default-resources) in all managed namespaces.
+    - Namespace Provisioner creates [default resources](default-resources.hbs.md) in all managed namespaces.
     - Namespace Provisioner creates additional Platform Operator templated resources stored in Git repository locations specified under the `additional_sources` section in Namespace Provisioner configuration. For more information, see [Customize Installation of Namespace Provisioner](customize-installation.md).
```

Comment:
Is this customize-installation.hbs.md ?

Reply:
As discussed with @kcoriordan , these changes in file extensions will be handled in a separate PR . Closing this comment